sources/packages/web/src/components/institutions/active-application/ActiveApplicationSummaryData.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+  <body-header
+    title="Applications"
+    :recordsCount="applications.results?.length"
+    class="m-1 mb-8"
+  >
+    <template #actions>
+      <v-text-field
+        class="v-text-field-search-width"
+        density="compact"
+        label="Search name or application #"
+        variant="outlined"
+        v-model="searchCriteria"
+        @keyup.enter="searchActiveApplications"
+      >
+        <template v-slot:prependInner>
+          <font-awesome-icon :icon="['fas', 'search']" class="m" />
+        </template>
+      </v-text-field>
+    </template>
+  </body-header>
+  <content-group>
+    <toggle-content :toggled="!applications.results?.length">
+      <DataTable
+        :value="applications.results"
+        :lazy="true"
+        class="p-m-4"
+        :paginator="true"
+        :rows="pageLimit"
+        :rowsPerPageOptions="PAGINATION_LIST"
+        :totalRecords="applications.count"
+        @page="pageEvent"
+        @sort="sortEvent"
+      >
+        <template #empty>
+          <p class="text-center font-weight-bold">No records found.</p>
+        </template>
+        <Column field="fullName" header="Name" :sortable="true"> </Column>
+        <Column field="studyStartPeriod" header="Study dates">
+          <template #body="slotProps">
+            <span>
+              {{ dateString(slotProps.data.studyStartPeriod) }} -
+              {{ dateString(slotProps.data.studyEndPeriod) }}
+            </span>
+          </template>
+        </Column>
+        <Column
+          field="applicationNumber"
+          header="Application #"
+          :sortable="true"
+          headerStyle="width: 20%"
+        ></Column>
+        <Column field="applicationStatus" header="Status">
+          <template #body="slotProps">
+            <StatusChipActiveApplication
+              :status="slotProps.data.applicationSholasticStandingStatus"
```

Comment:
Please check the typos in all the places that have `applicationSholasticStandingStatus`.